libportability/src/lib.rs

```diff
@@ -1123,7 +1127,15 @@ pub extern "C" fn vkCmdCopyImage(
     regionCount: u32,
     pRegions: *const VkImageCopy,
 ) {
-    unimplemented!()
+    gfxCmdCopyImage(
```

Comment:
I believe we should be auto-generating all that `extern -> gfx` pass-through logic, at least